schema.sql

```diff
@@ -12,3 +14,21 @@ CREATE TABLE animals(
    PRIMARY KEY(id)      
 );
 
+
+CREATE TABLE owners(
+   id INT GENERATED ALWAYS AS IDENTITY,
+   full_name    VARCHAR(255),
+   age INT,
+   PRIMARY KEY(id)      
+);
+
+CREATE TABLE species(
+   id INT GENERATED ALWAYS AS IDENTITY,
+   name    VARCHAR(255),
+   PRIMARY KEY(id)      
+);
+
+ALTER TABLE animals
+DROP COLUMN species,
+ADD species_id INT,
+ADD owner_id INT,
```

Comment:
- [ ] According to the requirements kindly add foreign key referencing to the table specified. The requirements are below:

- Add column species_id which is a foreign key referencing species table

- Add column owner_id which is a foreign key referencing the owners table